models/src_recharge.yml

```diff
@@ -425,6 +425,51 @@ sources:
           - name: _fivetran_synced
             description: "{{ doc('_fivetran_synced') }}"
 
+      - name: orders # applies to connectors created on or after 2024-06-18
```

Comment:
@fivetran-jamie I hand't considered that, so I'm flexible either way! Could you explain the benefits?

Reply:
Great question! Not much since we don't have any freshness tests in this package haha

Only benefit I can think of is getting rid of a floating `orders` source entity in the DAG for people who don't have it yet. Not super high priority, especially given that there's not an easy way to get rid of a floating `order` source in the DAG for people with `orders`.

Honestly let's leave it as is. I feel like it's more beneficial to potentially call people's attention to the upcoming `ORDERS` transition